core/camel-core-languages/src/main/docs/modules/languages/pages/simple-language.adoc

```diff
@@ -160,18 +160,18 @@ converted message can be null.
 |env.foo |String |refer to the system environment variable
 
 |exception |Object |refer to the exception object on the exchange, is *null* if
-no exception set on exchange. Will fallback and grab caught exceptions
+no exception set on exchange. Will fall back and grab caught exceptions
```

Comment:
Maybe it can be rewritten in better English:

> refer to the exception object on the exchange, which will be *null* if no exception is set on the exchange. If no exception is found, it will fall back to retrieve any caught exceptions (`Exchange.EXCEPTION_CAUGHT`) present on the Exchange.

and with the same wording for the `exception.message` and `exception.stacktrace` below.